### DISCUSSION TITLE
Can't change the aspect ratio on iOS

### DISCUSSION THREAD

**Discussion by Krillko** [mebjas/html5-qrcode#894] [Q&A]
I'm trying to limit the height of video preview but it's always comes out slightly higher than wide.

example: 
```
const config = { aspectRatio: 2, fps: 10, qrbox: { width: 350, height: 250 } };
```
Works fine on desktop chrome, but does nothing on iOS.
(I don't have an android to test with)

I've also tried using css to max height it, but that seems to break it completely.

**Comment by ROBERT-MCDOWELL** [comment]
why aspect ratio 2???

**Reply by Krillko** [reply]
According to typescript aspectRatio is supposed to be a number, and it works in chrome, translates to "2:1"

**Comment by ROBERT-MCDOWELL** [comment]
ok and write after aspect ratio 2.0 width: 350, height: 250 does not make sens at all as 350x250 is not aspect ratio 2.0. chose one or the other and for sure you will have a more accurate result.

**Reply by Krillko** [reply]
I'm trying to change the size of the entire video-preview, `qrbox` controls the box in the middle


![Skärmbild_2024-02-28_10_17](https://github.com/mebjas/html5-qrcode/assets/5945212/08b03a05-25a0-4e96-8c69-8ed93dd54183)

**Comment by ROBERT-MCDOWELL** [comment]
search on the closed issues, your case has been solved many times already and it's not my duty to search for you.
